python/cudnn/sdpa/bwd/engines.py

```
@@ -0,0 +1,374 @@
+
+def engine_name(arch: str = "sm120") -> str:
+    """The shipped engine name for a coverage cell (test/user convenience)."""
+
+    return f"sdpa_bwd_{arch}"
+
+
+# Preference order: the ranked plan list offers these in this order (see
+# cudnn/sdpa/bwd/engine.py, which wraps each spec as a BaseEngine).
+ENGINE_SPECS = (_sm120_spec(),)
+
+__all__ = ["Capabilities", "EngineSpec", "ENGINE_SPECS", "SdpaBwdKnobs", "analyze_for", "engine_name", "mismatch"]
```

Ruff (0.16.1)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)